### PR TITLE
feature/legal-modal-2

### PR DESCRIPTION
**Pull request by Cardamced**
## Description

Please include a summary of the change or which issue is fixed.

## Type of change

- [X] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Area of change

- [X] Frontend
- [ ] Backend
- [ ] Other: .....

## General checklist:

- [X] My code follows the style guidelines of this project
- [X] I ran `bun run format`/`bun run lint` before commit
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation where needed
- [X] My changes generate no new warnings
- [X] I have checked my code and corrected any misspellings
- [X] I have updated `develop` and merged to my branch before submitting pull request
- [X] My pull request generate no conflicts with `develop` branch
- [X] I requested code review from other team members

## Frontend checklist:

- [X] I followed guidelines for `HTML`/`JSX`/`Typescript`
- [X] My `Javascript` generate no new console errors
- [X] I tested my code cross browsers
- [X] My slice is pixel perfect for both desktop and mobile according to design
- [ ] I conducted basic QA to assure all features are working
- [X] I tested responsive for mobile and tablet resolutions

## Backend checklist:

- [ ] I tested admin by manually adding content from zero
- [ ] I created easy to use admin experience which is self-explanatory
- [ ] I added description to admin fields in hard-to-understand areas
- [ ] I followed guidelines for naming `Typescript` variables
- [ ] I conducted basic QA to assure all features are working

### CODE REVIEW

**Review by No3ml3 — DISMISSED** [review]

**Review by Justicesvr — DISMISSED** [review]

File: packages/frontend/web/src/pages/LegalModal.tsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import { useEffect, useState } from 'react';
 
 export default function LegalNotice() {
   const [isModalShown, setIsModalShown] = useState(true);
+  const now: Date = new Date();
```
Comment (YDrogen) [inline packages/frontend/web/src/pages/LegalModal.tsx]: Peut être sorti du component

**Review by YDrogen — CHANGES_REQUESTED** [review]
Le nom de la branche ?
Pourquoi hotfix ? C'est un problème qui se trouve sur main ?
Pourquoi du pascal case ?

File: packages/frontend/web/src/pages/LegalModal.tsx (hunk 1 above)
Comment (Cardamced) [reply packages/frontend/web/src/pages/LegalModal.tsx]: Done. Ça va finir par rentrer ces const date de portée générale...

Comment (Cardamced) [reply packages/frontend/web/src/pages/LegalModal.tsx]: Le nom de la branche c'est corrigé.

**Review by No3ml3 — APPROVED** [review]

**Review by jaalazard — APPROVED** [review]

File: packages/frontend/web/src/pages/LegalModal.tsx (hunk 1 above)
Comment (YDrogen) [reply packages/frontend/web/src/pages/LegalModal.tsx]: J'attends la bonne PR du coup

Comment (Cardamced) [reply packages/frontend/web/src/pages/LegalModal.tsx]: Je ne suis pas sûr de bien comprendre... Je ferme celle-ci ; je recrée une PR avec un nom correct ? Parce que l'autre feature/legalmodal existait déjà et avait été mergée. Mais peut-être que je comprends mal.
Pas pigé non plus la relation hotfix / main... rien d'écrit dans le readme. Mais peut-être que c'est plus simple de faire un point sur le nommage en vis-à-vis en 2 sec...

Comment (YDrogen) [reply packages/frontend/web/src/pages/LegalModal.tsx]: Je ne vais pas merge cette PR, la branche n'a pas le bon nom

Comment (Cardamced) [reply packages/frontend/web/src/pages/LegalModal.tsx]: Ben ça je l'avais compris ^^

**Pull request closed without merging**